src/test/java/PlayListTests.java

```diff
@@ -66,11 +58,11 @@ public void createNameTwice () {
         playListPage.addingPlaylistbtn();
         playListPage.enterPlayListName("second");
         //EXPECTED TO BE FALSE
-        Assert.assertEquals(playListPage.getConfirmationMessage(),false,"CANNOT ENTER SAME NAME");
+        Assert.assertEquals(playListPage.getConfirmationMessage(), false, "CANNOT ENTER SAME NAME");
     }
 
     @Test(enabled = true, priority = 3)
-    public void createNameWith10Chars ()  {
+    public void createNameWith10Chars() {
```

Comment:
Test failed!

You got this problem, please correct it.

![image](https://user-images.githubusercontent.com/104761482/211770504-de782382-b1c1-4f2b-87df-0bb45e6383d7.png)

Reply:
I fixed this one-remove letter a of first word. Please check.